worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/DiskOptimizedClipboard.java

```diff
@@ -336,13 +336,13 @@ private void closeDirectBuffer(ByteBuffer cb) {
     }
 
     @Override
+    @SuppressWarnings("ResultOfMethodCallIgnored")
     public void close() {
         try {
             if (byteBuffer != null) {
                 byteBuffer.force();
                 fileChannel.close();
                 braf.close();
-                //noinspection ResultOfMethodCallIgnored
                 file.setWritable(true);
```

Comment:
It might be worth throwing an exception ourselves if we cannot set the file as writeable

Reply:
We do so a bit further down the line